src/Microsoft.PowerShell.Commands.Utility/commands/utility/CSVCommands.cs

```diff
@@ -96,7 +96,7 @@ public virtual void WriteCsvLine(string line)
         /// </summary>
         protected override void BeginProcessing()
         {
-            if (this.MyInvocation.BoundParameters.ContainsKey("IncludeTypeInformation") && this.MyInvocation.BoundParameters.ContainsKey("NoTypeInformation"))
+            if (this.MyInvocation.BoundParameters.ContainsKey(nameof(IncludeTypeInformation)) && this.MyInvocation.BoundParameters.ContainsKey(nameof(NoTypeInformation)))
```

Comment:
Can we use different parameter sets to exclude the code?

Reply:
@iSazonov we already discussed that here 

https://github.com/PowerShell/PowerShell/pull/5164#discussion_r145791740

The parameter sets are currently being used for delimiter vs culture. That would require we add 2 more parameter sets. It would make the code harder to maintain. The parameter sets would be confusing to the user because we are hiding `-NoTypeInformation` with `DontShow=true`. @SteveL-MSFT  agreed that the runtime check was fine for now until we could address the larger general problem of mutex parameters. which prompted issue #5175

Reply:
Sorry, I skipped this.